src/example_package/gui/kivy/app.kv

```diff
@@ -0,0 +1,175 @@
+<MainScreen>:
+    orientation: 'horizontal'
+    padding: 40
+    spacing: 40
+    canvas.before:
+        Color:
+            rgba: 0.925, 0.925, 0.925, 1
+        Rectangle:
+            size: self.size
+            pos: self.pos
+    Menu:
+        id: menu
+    Content:
+        id: content
+
+<Menu>:
+    orientation: 'vertical'
+    size_hint_x: 0.3
+    MenuItem:
+        text1: 'Quick Start'
+        text2: 'Instructions on how to quickly start using the app.'
```

Comment:
Each menu item would have to have something like `menu_item_enum_name` string property